### ISSUE TITLE
WCAG requirements are not met

### ISSUE THREAD

**Issue by haattis** [itforeninger/itforeninger.no#18]
The following text fragments on the main page does not meet WCAG requirements:
- Header - "FIF ER"
- Header - "BLI MED I FIF"
- paragraph - "kontakt@itforeninger.no"
- Header (and arrow) - "LES OM SAKENE VI JOBBER MED"

Find suitable colors that meet the requirements. Note that these colors are currently used elsewhere, including in the dark theme.
